hw/ip/otbn/dv/model/otbn_trace_checker.cc

```diff
@@ -228,14 +229,18 @@ const OtbnIssTraceEntry::IssData *OtbnTraceChecker::PopIssData() {
   return &last_data_;
 }
 
+void OtbnTraceChecker::set_no_sec_wipe_chk(bool new_val) {
+  no_sec_wipe_data_chk = new_val;
+}
+
 bool OtbnTraceChecker::MatchPair() {
   if (!(rtl_pending_ && iss_pending_)) {
     return true;
   }
   rtl_pending_ = false;
   iss_pending_ = false;
   iss_started_ = false;
-  if (!(rtl_entry_.compare_rtl_iss_entries(iss_entry_))) {
+  if (!(rtl_entry_.compare_rtl_iss_entries(iss_entry_, no_sec_wipe_data_chk))) {
```

Comment:
I think you still need to clear the flag again at the end of this function if this was a wipe entry?